clients/wallet/src/stellar_wallet.rs

```diff
@@ -141,9 +157,26 @@ impl StellarWallet {
 			is_public_network,
 			transaction_submission_lock: Arc::new(Mutex::new(())),
 			cache,
+			max_retry_attempts_before_fallback: Self::MAX_RETRY_ATTEMPTS_BEFORE_FALLBACK,
+			max_backoff_delay: Self::MAX_BACKOFF_DELAY,
 		})
 	}
 
+	pub fn with_max_retry_attempt_before_fallback(mut self, max_retries: u8) -> Self {
+		self.max_retry_attempts_before_fallback = max_retries;
+
+		self
+	}
+
+	pub fn with_max_backoff_delay(mut self, max_backoff_delay: u8) -> Self {
+		// a number more than the default max would be too large
+		if max_backoff_delay < Self::MAX_BACKOFF_DELAY {
+			self.max_backoff_delay = max_backoff_delay;
+		}
```

Comment:
I think this check can be removed if we change the max_backoff_delay the way I described. Just saying, so we don't forget

Reply:
hhm, user could set it to `u16::MAX`.  Maybe it's best to let the user allow it,  only if it's not too big.

And the const `DEFAULT_MAX_BACKOFF_DELAY_IN_SECS` is found in the `stellar_wallet`, not in horizon.